# Kashubian translation for linuxmint
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2016-11-02 13:58+0000\n"
"PO-Revision-Date: 2010-05-19 09:21+0000\n"
"Last-Translator: Launchpad Translations Administrators <Unknown>\n"
"Language-Team: Kashubian <csb@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2017-06-28 07:44+0000\n"
"X-Generator: Launchpad (build 18416)\n"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:111
#: usr/lib/linuxmint/mintbackup/mintbackup.py:251
#: usr/lib/linuxmint/mintbackup/mintbackup.py:443
#: usr/lib/linuxmint/mintbackup/mintbackup.py:461
#: usr/lib/linuxmint/mintbackup/mintbackup.py:527
#: usr/lib/linuxmint/mintbackup/mintbackup.py:530
#: usr/lib/linuxmint/mintbackup/mintbackup.py:582
#: usr/lib/linuxmint/mintbackup/mintbackup.py:585
#: usr/lib/linuxmint/mintbackup/mintbackup.py:605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:628
#: usr/lib/linuxmint/mintbackup/mintbackup.py:635
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1499
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1597
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1661 generate_desktop_files.py:26
#: generate_desktop_files.py:41
msgid "Backup Tool"
msgstr "Nôrzãdze zazychrowaniô pòdôwków"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:168
msgid "Preserve structure"
msgstr "Ùtrzëmôj strukturã"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:170
msgid ".tar file"
msgstr "lopk .tar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:171
msgid ".tar.bz2 file"
msgstr "lopk .tar.bz2"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:172
msgid ".tar.gz file"
msgstr "lopk .tar.gz"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:178
msgid "Never"
msgstr "Nigdë"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:179
msgid "Size mismatch"
msgstr "Lëchô miara"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:180
msgid "Modification time mismatch"
msgstr "Czas mòdifikacëji nie pasëje"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:181
msgid "Checksum mismatch"
msgstr "Kòntrolowóne zesëmòwanié nie pasëje"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:182
msgid "Always"
msgstr "Wiedno"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:204
msgid "Excluded paths"
msgstr "Wëłączoné stegnë"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:214
msgid "Type"
msgstr "Ôrt"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:218
msgid "Detail"
msgstr "Detale"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:224
#: usr/lib/linuxmint/mintbackup/mintbackup.py:233
msgid "Path"
msgstr "Stegna"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:227
#: usr/lib/linuxmint/mintbackup/mintbackup.py:236
msgid "Error"
msgstr "Fela"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:268
msgid "Store?"
msgstr "Ùtrzëmac?"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:271
#: usr/lib/linuxmint/mintbackup/mintbackup.py:284
msgid "Name"
msgstr "Miono"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:281
msgid "Install"
msgstr "Instalëjë"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:289
msgid "Backup or Restore"
msgstr "Kòpijô bezpiekù ë doprowôdzanié nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:290
msgid "Choose from the following options"
msgstr "Wëbierzë jedną z nôslédnych òptacëjów"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:292
#: usr/lib/linuxmint/mintbackup/mintbackup.py:303
#: usr/lib/linuxmint/mintbackup/mintbackup.py:317
#: usr/lib/linuxmint/mintbackup/mintbackup.py:324
#: usr/lib/linuxmint/mintbackup/mintbackup.py:328
#: usr/lib/linuxmint/mintbackup/mintbackup.py:333
msgid "Backup files"
msgstr "Kòpijô bezpiekù lopków"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:293
#: usr/lib/linuxmint/mintbackup/mintbackup.py:337
#: usr/lib/linuxmint/mintbackup/mintbackup.py:347
#: usr/lib/linuxmint/mintbackup/mintbackup.py:353
#: usr/lib/linuxmint/mintbackup/mintbackup.py:358
msgid "Restore files"
msgstr "Doprowôdzë lopczi nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:294
#: usr/lib/linuxmint/mintbackup/mintbackup.py:362
#: usr/lib/linuxmint/mintbackup/mintbackup.py:367
#: usr/lib/linuxmint/mintbackup/mintbackup.py:373
#: usr/lib/linuxmint/mintbackup/mintbackup.py:378
msgid "Backup software selection"
msgstr "Ùsôdzë kòpijã bezpiekù wëbróny softwôrë"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:295
#: usr/lib/linuxmint/mintbackup/mintbackup.py:382
#: usr/lib/linuxmint/mintbackup/mintbackup.py:387
#: usr/lib/linuxmint/mintbackup/mintbackup.py:394
msgid "Restore software selection"
msgstr "Doprowôdzë wëbróną softwôrã nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:297
msgid "Make a backup of your files"
msgstr "Ùsôdzë kòpiją bezpiekù swòjich lopków"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:298
msgid "Save the list of installed applications"
msgstr "Zapiszë lëstã winstalowónych aplikacëjów"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:299
msgid "Restore a previous backup"
msgstr "Doprowôdzë pòprzédną kòpijã bezpiekù nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:300
msgid "Restore previously installed applications"
msgstr "Doprowôdzë pòprzédno winstalowóné aplikacëje nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:304
msgid "Please select a source and a destination for your backup"
msgstr "Wëbierzë proszã zdrój abò plac dlô twòji Kòpiji bezpiekù"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:305
#: usr/lib/linuxmint/mintbackup/mintbackup.py:341
#: usr/lib/linuxmint/mintbackup/mintbackup.py:349
msgid "Source:"
msgstr "Zdrój:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:306
#: usr/lib/linuxmint/mintbackup/mintbackup.py:342
msgid "Destination:"
msgstr "Cél:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:307
#: usr/lib/linuxmint/mintbackup/mintbackup.py:343
msgid "Advanced options"
msgstr "Awansowóné òptacëje"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:308
#: usr/lib/linuxmint/mintbackup/mintbackup.py:350
msgid "Description:"
msgstr "Òpisënk:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:309
msgid "Output:"
msgstr "Wińdzenié:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:310
#: usr/lib/linuxmint/mintbackup/mintbackup.py:344
msgid "Overwrite:"
msgstr "Nadpisëwanié:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:311
msgid "Confirm integrity"
msgstr "Pòcwierdzë integralnotã"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:312
msgid "Follow symlinks"
msgstr "Biéj za symbòlicznyma lënkama"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:313
msgid "Preserve permissions"
msgstr "Ùtrzëmôj prawa przistãpù"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:314
msgid "Preserve timestamps"
msgstr "Ùtrzëmôj znakòwniczi czasu"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:318
msgid "Please select any files or directories you want to exclude"
msgstr ""
"Wëbierzë proszã swòjowólné lopczi abò katalodżi, chtërné chcesz wëkluczëc"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:319
msgid "Exclude files"
msgstr "Wëłączoné lopczi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:320
msgid "Exclude directories"
msgstr "Wëkluczë katalodżi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:321
msgid "Remove"
msgstr "Rëmôj"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:325
msgid "Please review the information below before starting the backup"
msgstr ""
"Przezdrzi proszã ùniższą wëdowiédzã, niglë naczniesz ùsôdzac kòpijã bezpiekù"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:329
msgid "Please wait while your files are being backed up"
msgstr "Proszã żdac przë ùsôdzaniu kòpiji bezpiekù twòjich lopków"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:330
#: usr/lib/linuxmint/mintbackup/mintbackup.py:375
msgid "Backing up:"
msgstr "Ùsôdzanié kòpiji bezpiekù:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:334
#: usr/lib/linuxmint/mintbackup/mintbackup.py:379
msgid "The backup is now finished"
msgstr "Ùsôdzanié kòpiji bezpiekù ùdało zakùńczoné"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:338
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr ""
"Wëbierzë proszã ôrt kòpiji bezpiekù, chtërną chcesz doprowôdzëc nazôd, ji "
"pòłożenié ë przeznaczony plac"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:339
msgid "Archive"
msgstr "Archiwa"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:340
msgid "Directory"
msgstr "Katalog"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:348
msgid "Please review the information below"
msgstr "Przezdrzi proszã ùniższą wëdowiédzã"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:354
msgid "Please wait while your files are being restored"
msgstr "Proszã żdac przë doprowôdzaniu twòjich lopków nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:355
msgid "Restoring:"
msgstr "Doprowôdzanié nazôd:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:359
msgid "The restoration of the files is now finished"
msgstr "Doprowôdzanié lopków nazôd ùdało zakùńczoné"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:363
msgid "Please choose a destination"
msgstr "Proszã wëbrac cél"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:364
#: usr/lib/linuxmint/mintbackup/mintbackup.py:546
msgid "Destination"
msgstr "Cél"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:368
msgid "The list below shows the packages you added to Linux Mint"
msgstr "Ùniższô lësta wëskrzëniwô paczétë, chtërné môsz dodóné do Linux Mint"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:369
#: usr/lib/linuxmint/mintbackup/mintbackup.py:389
msgid "Select all"
msgstr "Wëbierzë wszëtkò"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:370
#: usr/lib/linuxmint/mintbackup/mintbackup.py:390
msgid "Deselect all"
msgstr "Òdznaczë wszëtkò"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:374
msgid "Please wait while your software selection is being backed up"
msgstr "Proszã żdac, jak je ùsôdzanô kòpijô bezpiekù wëbróny softwôrë"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:383
msgid "Please select a saved software selection"
msgstr "Wëbierzë proszã zapisôny wëbiérk softwôrë"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:384
msgid "Software selection:"
msgstr "Wëbiérk softwôrë:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:388
msgid "Select the packages you want to install"
msgstr "Wëbierzë paczétë, chtërné chcesz wistalowac"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:391
msgid "Refresh"
msgstr "Zladëjë znowa"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:395
msgid "The restoration is now finished"
msgstr "Zakùńczono doprôwdzanié nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:396
msgid "Your package selection was restored successfully"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:452
#: usr/lib/linuxmint/mintbackup/mintbackup.py:470
msgid "Invalid path"
msgstr "Lëchô stegna"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:452
#: usr/lib/linuxmint/mintbackup/mintbackup.py:470
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s nie je w Twòjim zdrojowim katalogù. Nie òstôł dodóny."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:527
msgid "Please choose directories for the source and the destination"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:530
msgid ""
"Please choose different directories for the source and the destination"
msgstr "Wëbierzë proszã wszelejaczé katalodżi dlô zdroju ë docélowégò placu"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:545
msgid "Source"
msgstr "Zdrój"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:548
msgid "Description"
msgstr "Òpisënk"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:552
msgid "Compression"
msgstr "Kòmpresëjô"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:556
msgid "Overwrite destination files"
msgstr "Nadpiszë docélowé lopczi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:559
msgid "Exclude"
msgstr "Wëłączë"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:582
msgid "Please choose a file to restore from"
msgstr "Wëbierzë proszã lopk, z chtërnegò bãdzesz doprowôdzac nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:585
#: usr/lib/linuxmint/mintbackup/mintbackup.py:605
msgid "Please choose a destination directory"
msgstr "Wëbierzë proszã docélowi katalog"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:628
msgid "No packages need to be installed at this time"
msgstr "W tim czasu nie dô niżódnegò nótérnegò paczétu do winstalowaniô"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:635
msgid "Please select one or more packages to install"
msgstr "Wëbierzë proszã jeden abò wicy paczétów do winstalowaniô"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:703
#: usr/lib/linuxmint/mintbackup/mintbackup.py:704
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1206
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1208
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1514
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1515
msgid "Calculating..."
msgstr "Rechòwanié..."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:719
#, python-format
msgid "Skipping %(skipped_file)s because named pipes are not supported."
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:761
#: usr/lib/linuxmint/mintbackup/mintbackup.py:801
msgid "Skipping broken link"
msgstr "Przepùszczanié lëchegò lënka"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:762
msgid "Broken link"
msgstr "Lëchi lënk"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:765
#: usr/lib/linuxmint/mintbackup/mintbackup.py:805
msgid "Skipping link"
msgstr "Przepùsczë lënk"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:829
#: usr/lib/linuxmint/mintbackup/mintbackup.py:838
#: usr/lib/linuxmint/mintbackup/mintbackup.py:847
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1267
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1278
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1291
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1355
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1364
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1374
msgid "Skipping identical file"
msgstr "Przepùszczanié równegò lopka"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:869
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Bôczënk: Niejedné lopczi nie òstałé zapisóné, skòpérowóno: %(current_file)d "
"lopków z całowny wielënë %(total)d"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:873
msgid "An error occurred during the backup"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:883
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1560
msgid "The backup was aborted"
msgstr "Przerwóno ùsôdzanié kòpiji bezpiekù"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:891
msgid "The backup completed successfully"
msgstr "Ùdałé ùsôdzenié kòpiji bezpiekù"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:982
msgid "Checksum Mismatch:"
msgstr "Kòntrolné zesëmòwanié nie pasëje:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:983
msgid "Checksum Mismatch"
msgstr "Kòntrolné zesëmòwanié nie pasëje"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1035
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1037
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1066
msgid "Calculating checksum"
msgstr "Rechòwanié kòntrolnegò zesëmòwaniô"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1112
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1136
msgid "(Not created with the Backup Tool)"
msgstr "(Nie ùsôdzoné przez nôrzãdze kòpiji bezpiekù)"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1398
#, python-format
msgid ""
"Warning: Some files were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1401
msgid "An error occurred during the restoration"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1412
msgid "The restoration was aborted"
msgstr "Òprzestóné doprowôdzanié nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1420
msgid "The restoration completed successfully"
msgstr "Ùdałé doprowôdzanié nazôd"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1551
msgid "An error occurred during the backup:"
msgstr "Fela przë ùsôdzaniu zazychrowaniô pòdôwków:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1568
msgid "Your software selection was backed up successfully"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1597
msgid "The specified file is not a valid software selection"
msgstr "Nacéchòwóny lopk nie je dobrim wëbierkã softwôrë"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1661
msgid "An error occurred while accessing the file"
msgstr "Pòkôza sã fela przistãpù do lopka"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1648
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1654
msgid "Could not locate the package"
msgstr "Ni mòże nalezc paczétu"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1686
msgid "Please wait, this can take some time"
msgstr "Proszã żdac, to mòże kąsk zbawic"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1688
msgid "The installation is complete"
msgstr "Instalacëjô zakùńczonô"

#: generate_desktop_files.py:26 generate_desktop_files.py:41
msgid "Make a backup of your home directory"
msgstr "Zazychrëjë pòdôwczi twòjegò domôcegò kataloga"

#~ msgid "Archiving your home directory..."
#~ msgstr "Archiwizowanié twòjegò domôcegò kataloga..."

#~ msgid "Restoration successful"
#~ msgstr "Ùdałé doprowôdzanié nazôd"

#~ msgid "Cleaning up..."
#~ msgstr "Czëszczenié..."

#~ msgid "Restoration failed"
#~ msgstr "Doprowôdzanié nazôd nie darzëło sã"

#~ msgid "Read error"
#~ msgstr "Fela czëtaniô"

#~ msgid "Exclude folders"
#~ msgstr "Wëłączoné katalodżi"

#~ msgid "Hidden paths"
#~ msgstr "Zataconé stegnë"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr "<big><b>Ladëjë pòdôwczi do domôcegò kataloga</b></big>"

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "nie je w domôcym katalogù. Ni dodóné."

#~ msgid "Excluded Files and Directories"
#~ msgstr "Wëłączoné lopczi ë katalodżi"

#~ msgid "Restore"
#~ msgstr "Doprowôdzë nazôd"

#~ msgid "Overwrite existing files"
#~ msgstr "Nadpiszë lopczi"

#~ msgid "View content"
#~ msgstr "Òbôczë zamkłosc"

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "Zazychrowanié pòdôwków domôcegò kataloga òsta ùdało zapisónô do"

#~ msgid "Backup failed"
#~ msgstr "Zazychrowanié pòdôwków ni òstało zapisóné"

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr ""
#~ "Rozpakòwëwanié zamkłoscë zazychrowaniô pòdôwków do domôcegò kataloga..."

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "Kòpérowanié lopka zazychrowaniô pòdôwków do domôcegò kataloga..."

#~ msgid "Backup successful"
#~ msgstr "Zazychrowanié pòdôwków ùdało zakùńczonôé"

#~ msgid "Your backup was successfully restored"
#~ msgstr "Zazychrowanié pòdôwków ùdało doprowôdzoné nazôd"

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr "Fela przë doprowôdzaniu zazychrowaniô pòdôwków nazôd:"

#~ msgid "Opening the backup archive..."
#~ msgstr "Òtmëkanié archiwùm zazychrowaniô pòdôwków..."

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "Fela przë òtmëkaniu zazychrowaniô pòdôwków:"

#~ msgid "Restore your personal data from this backup"
#~ msgstr "Doprowôdzë swòje pòdôwczi z negò zazychrowaniô nazôd"

#~ msgid "Included hidden directories"
#~ msgstr "Dołączoné zataconé katalodżi"

#~ msgid "Included"
#~ msgstr "Dołączoné"

#~ msgid "Backup"
#~ msgstr "Zazychrowanié pòdôwków (backup)"
